packages/documentation-site/patternfly-docs/content/design-guidelines/styles/theming/theming.md

```diff
@@ -0,0 +1,80 @@
+</div>
+
+### Dark mode
+
+In dark mode, light text is presented on dark backgrounds backgrounds, and our color palette adapts to maintain a [text contrast ratio of at least 4.5:1](https://www.w3.org/WAI/WCAG22/quickref/?versions=2.1#contrast-minimum) and [non-text contrast ratio of at least 3:1](https://www.w3.org/WAI/WCAG22/quickref/?versions=2.1#non-text-contrast). Some users might prefer dark mode for aesthetics, while others find it to be easier on the eyes and less straining for those with light sensitivities. 
+
+<div class="ws-docs-content-img" style="width:90%">
+![A collage of multiple dark-themed PatternFly components, like a calendar, alert, and menu. The components have dark gray backgrounds and white text.](./img/components-dark.svg)
+</div>
+
+### High contrast mode
+
+<Alert 
+variant="info" 
+title="Beta feature"
+actionLinks={
+<AlertActionLink component="a" href="MEDIUM ARTICLE">
+Learn more
+</AlertActionLink>
+}
+>
+<p> High contrast mode is still under development and will continue to evolve and be enabled for charts and extensions. This beta allows you to preview our progress. </p>
+</Alert>
+
+High contrast mode adjusts our default colors to meet an [enhanced contrast ratio of at least 7:1](https://www.w3.org/WAI/WCAG21/Understanding/contrast-enhanced.html), making it more suitable for users who require higher contrast between UI elements. By using wider border strokes and adjusted fill colors, high contrast mode creates more visual distinction and clarity between interactive elements.
+
+In high contrast mode, distinct borders are also added to components to ensure that their boundaries are clearly defined without requiring users to rely on subtle background colors as a visual cue. 
+
+## Using themes in Figma
+
+Our Figma libraries fully support theming. Designers can create a single layout and then use Figma's appearance feature to swap a wireframe to light, dark, or high contrast mode. This makes it easy to visually test and validate designs across all supported themes.
+
+<div class="ws-docs-content-img">
+![A dark colored flyout menu within an "Appearance" section. The menu expands "Semantic Color Tokens" with selectable options for "Auto (Light)", "Light", and "Dark."](./img/figma-dark-mode.png)
+</div>
+
+## Custom themes 
+
+To branch off of our themes and create your own, you can alter design token values to specify new styles. 
+
+Our layered design tokens allow you to create new themes simply by changing base token values in a single file. These adjustments will then apply appropriately to all semantic tokens across your codebase. 
+
+### When to create a custom theme 
+
+| **Don't** | **Do** |
+| --- | --- |
+| Create a theme just for one-off adjustments, like changing a single button color. Instead, modify small design changes using component props or utility classes. | Create a theme to adjust the overall brand identity of your application. |
```

Comment:
Is it still valid to just "modify" a theme (with one-off adjustments) - you just wouldn't call it a theme? Or do we not encourage modifying themes?

I ask this because I'm thinking about a "how to theme/customize patternfly" doc (from the developer's perspective) with the upcoming theme builder, and would think that customizations to spacers, font-sizes, colors, etc could happen without essentially re-branding PF with a whole new theme.